cucumber/authenticators_jwt/features/authn_jwt_input_validation.feature

```diff
@@ -0,0 +1,108 @@
+Feature: JWT Authenticator - Input Validation
+
+  Background:
+    Given I initialize JWKS endpoint with file "myJWKs.json"
+    And I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on raw JWKs.
+
+      - !variable
+        id: jwks-uri
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+    - !user
+      id: myuser
+      annotations:
+        authn-jwt/raw/project-id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/users
+      member: !user myuser
+    """
+    And I am the super-user
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+
```

Comment:
Do we need to add here : 
` And I successfully set authn-jwt "token-app-property" variable to value "host"`

Reply:
No. In this file all tests are identity from url . But anyway authentication don't comes to this step because it fails on input validation at the beggining of the flow